TaskManagementSystem/main.go

```diff
@@ -7,27 +7,58 @@ import (
 	"time"
 )
 
-func session2() {
-	var ProjectStatus string = "in progress"
+func TaskUpdate(CompletedTasks int) {
+
 	const NumberOfTotalTasks int = 100
-	CompletedTasks := 25
-	const (
-		Low = iota
-		Medium
-		High
-	)
-	var ProjectCompletionStatus bool = true
+	var ProjectStatus string
+
+	RemainingTasks := NumberOfTotalTasks - CompletedTasks
+	if CompletedTasks != NumberOfTotalTasks {
+		ProjectStatus = "in progress"
+	} else {
+		ProjectStatus = "finished"
+	}
+
+	var ProjectCompletionStatus bool
 	if ProjectStatus == "in progress" {
 		ProjectCompletionStatus = false
 	} else {
 		ProjectCompletionStatus = true
 	}
-	fmt.Println("Current project status:", strings.ToUpper(ProjectStatus))
-	fmt.Printf("Tasks completed: %d out of %d\n", CompletedTasks, NumberOfTotalTasks)
+
+	const (
+		Low = iota + 1
+		Medium
+		High
+	)
```

Comment:
We can remove constant, if you will receive the suggestion to move to global scope
```suggestion

```